Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -421,6 +421,147 @@ public IEnumerator NearInteractionTouchableUnityUiToggle()
             UnityEngine.Object.Destroy(canvas.gameObject);
         }
 
+        /// <summary>
+        /// Test minimum touch distance.
+        /// </summary>
+        /// <remarks>
+        /// Near interaction uses two separate distance calculations:
+        /// 1. The PokePointer uses the DistanceToTouchable to determine the closest touchable. This distance value is also used compared to PokePointer.TouchableDistance to determine the set of eligible touchables in the first place.
+        ///     There is no explicit spec which space this distance is computed in, although it should be in world space. Due to the way touchables convert pointer position into local space first and then do a scale-only inverse transform there may be errors.
+        /// 1. After the PokePointer has selected the closest touchable, it then performs a raycast against either the collider or the UnityUI canvas, depending on type. The (world space) length of that ray is then used as the actual distance for triggering touch events.
```

Comment:
Oh, i just meant to say that it's not guaranteed that the distance values returned by touchables match up with actual euclidian distance. It's probably fine as long as you don't combine rotations and non-uniform local scale.

A more annoying aspect is that the touchable distance function has nothing to do with the distances actually used for touch events. As long as the collider is reasonably flat and centered it works ok, but with larger colliders and possibly object centers outside the actual collider i would expect very weird results, because what is considered the "closest" touchable is based purely on the object transform while the raycast for events may hit a completely different object. That's not easy to fix without rewriting the whole system though.